Movies_MVVM/Modules/MovieDetailTab/View/Cells/CastCell/CastTableViewCell.swift

```diff
@@ -99,8 +81,18 @@ final class CastTableViewCell: UITableViewCell, UICollectionViewDataSource {
         setupConstraintsForCollectionView()
     }
 
-    private func showError(error: Error) {
-        print(error.localizedDescription)
+    private func updateView() {
+        viewModel?.successActorsHandler = {
+            DispatchQueue.main.async {
+                self.castCollectionView.reloadData()
+            }
+        }
+    }
+
+    private func showError() {
```

Comment:
```suggestion
    private func bind() {
```
все такие замыкания в такой метод собери